### PR TITLE
Add angular-scroll

### PR DESCRIPTION
**Pull request by quanack**
Adding library for Angular Scroll located at https://github.com/oblador/angular-scroll

### CODE REVIEW

File: ajax/libs/angular-scroll/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "git://git@github.com:oblador/angular-scroll.git"
+  },
+  "bugs": {
+    "url": "https://github.com/oblador/angular-scroll/issues"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://git@github.com:oblador/angular-scroll.git",
```
Comment (PeterDaveHello) [inline ajax/libs/angular-scroll/package.json]: Wrong URL format here.

File: ajax/libs/angular-scroll/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "basePath": "",
+    "files": [
```
Comment (PeterDaveHello) [inline ajax/libs/angular-scroll/package.json]: Would you please also add `angular-scroll.js` here?

**Comment by quanack** [comment]
Added angular-scroll.js
Fixed URL issue

**Comment by PeterDaveHello** [comment]
Please squash these commits into only one commit, thanks.

**Pull request closed without merging**